dune-project

```diff
@@ -53,9 +53,9 @@ possible and does not make any assumptions about IO.
   (pp (>= 1.1.2))
   (csexp (>= 1.4))
   (result (>= 1.5))
-  (ocamlformat-rpc-lib (and (>= 0.18.0) (< 0.19.0)))
+  (ocamlformat-rpc-lib (and (>= 0.19.0) (< 0.20.0)))
```

Comment:
Are we not compatible with 0.18 anymore?

Reply:
I think so:

![image](https://user-images.githubusercontent.com/16353531/133137671-8d9b7db6-812e-46ac-b7cf-866ace28d4ee.png)

Reply:
It's a little strange that an rpc library is going to be incompatible with a new version of OCaml. I wonder if they are just being conservative.

But even still, we still don't need the constraint. The solver will never select 0.18 for 4.13 anyway because of the constraint we inherit from ocamlformat-rpc-lib. Our constraints should only reflect interactions between our package and the dependencie, and not interactions between dependencies.